jwst/datamodels/schemas/ifucube.schema.yaml

```diff
@@ -72,4 +76,10 @@ allOf:
       default: 0.0
       ndim: 3
       datatype: float32
+    wavetable:
+      title: Wavelength value for slices
+      fits_hdu: WCS-TAB
```

Comment:
It might be useful to name this something more specific than WCS-TAB.  Maybe WAVETAB?  Indicate that is a tabular wavelength array.  One could also have tabular RA and DEC arrays.  The mapping of `wavetable <--> WAVETAB` is also more clear then.

Reply:
I believe the WCS-TAB is the fits standard. It what what the Spectral Fits paper used for the general case. WAVE-TAB is one the of WCS-TAB

Reply:
The EXTNAME value of "WCS-TAB" is not required by the FITS WCS standard for the xxxx-TAB algorithm. The only statement regarding the EXTNAME of the table HDU is "We recommend strongly that the value chosen for EXTNAME always begin with the four letters WCS-. If this is done, generic programs will recognize the table as part of the WCS portion of the data and will be less likely to separate the table from the rest of the WCS."

So in this case I agree with @jdavies-st that while EXTNAME='WCS-TAB' is perfectly acceptable given the FITS standard recommendations, I would suggest we change it to something a little more distinct from the "WAVE-TAB" value used for the CTYPEn keyword in the data HDU, just to avoid confusion when referring to their respective names. Perhaps something like "WCS-TABLE", as used in the example shown in the FITS WCS paper in Table 11.

Reply:
I again do not understand if the fits community " recommends strongly that the value chosen for EXTNAME always begin with the four letters WCS-. " why should we have it start with WAVE ?

Reply:
We aren't recommending starting with "WAVE". We're saying to use the recommended value of EXTNAME in the wavelength table HDU that starts with the recommended prefix "WCS-". We're just saying that the remaining suffix doesn't necessarily have to be "TAB". It could be anything we want and is not limited to the 8 character length of keyword names (this is a keyword **value**, not a name). So we could call it something like "WCS-ifunonlinearwavelengthtable" if we wanted to (although that would be very bad ...). We're just saying that an EXTNAME value of "WCS-TAB" is so similar to the required CTYPEn value of "WAVE-TAB", that there could be confusion when referring to one or the other. So make the EXTNAME value a little more distinct, such as "WCS-TABLE".